lib/ts/bwf_std_format.h

```diff
@@ -34,3 +34,17 @@ bwformat(ts::BufferWriter &w, ts::BWFSpec const &spec, std::atomic<T> const &v)
   return ts::bwformat(w, spec, v.load());
 }
 } // end namespace std
+
+namespace ts
+{
+namespace bwf
+{
+  struct Errno {
+    int _e;
+    Errno(int e) : _e(e) {}
```

Comment:
Should this converting constructor be declared `explicit`?

Reply:
Maybe. It's not expected to be used in a situation where that would make a difference (e.g., it would never be selected by passing `errno` to `BufferWriter::print`). I can do that for extra safety though.